File: bin/into-mongo.py
import pymongo
import json
import os

SERVER = 'localhost'
DATABASE = 'kevin'
COLLECTION = 'vcdb'

server = pymongo.Connection(SERVER)
db = server[DATABASE]
col = db[COLLECTION]


for (path, dirs, files) in os.walk('../vcdb'):
    for file in files:
        print('loading: '+os.path.join('../vcdb',file))
        infile = open(os.path.join('../vcdb',file), 'rb')
        incident = json.loads(infile.read())
        col.insert(incident)

File: bin/into-mongo.rb
require 'mongo'
require 'json'
require 'rubygems'
include Mongo

SERVER = 'localhost'
DATABASE = 'kevin'
COLLECTION = 'vcdb'

server = MongoClient.new(SERVER)
db = server.db(DATABASE)
col = db.collection(COLLECTION)

Dir.glob('../vcdb/*.json') do |filename|
    infile = open(filename, 'r')
    incident = JSON.parse(infile.read())
    col.insert(incident)
end

